packages/axios-extension/src/abap/abap-service-provider.ts

```diff
@@ -47,9 +46,7 @@ export class AbapServiceProvider extends ServiceProvider implements AbapServiceP
     public async getAtoInfo(): Promise<AtoSettings> {
```

Comment:
Should this also belong to the ADT service?

Reply:
Yes. This is also ADT service.